src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/ILogger/DatadogLoggingScope.cs

```diff
@@ -51,15 +58,19 @@ internal DatadogLoggingScope(Tracer tracer)
         public override string ToString()
         {
             var span = _tracer.ActiveScope?.Span;
+            if (span is null)
+            {
+                return _cachedFormat;
+            }
 
             return string.Format(
                 CultureInfo.InvariantCulture,
-                "dd_service:{0}, dd_env:{1}, dd_version:{2}, dd_trace_id:{3}, dd_span_id:{4}",
+                "{{\"dd_service\":\"{0}\", \"dd_env\":\"{1}\", \"dd_version\":\"{2}\", \"dd_trace_id\":\"{3}\", \"dd_span_id\":\"{4}\"}}",
```

Comment:
I'm not 100% sure if this is a good idea. It should help with log parsing, but also isn't exactly "standard". Any "good" `ILoggerProvider` implementation will be calling `GetEnumerator`, but that isn't guaranteed...

Reply:
Yeah I'm also not sure about that. What if the only change at this moment in time is to quote the property values? It's not as opinionated but I think it may be a small improvement over what we have today

Reply:
Yep, I'm more comfortable with that too. done in https://github.com/DataDog/dd-trace-dotnet/pull/1740/commits/1768e10368deb90ab9e43b8c98d22a4dadedf99d